.github/workflows/test.yaml

```diff
@@ -9,17 +9,20 @@ on:
   workflow_dispatch:
   schedule:
     - cron: '0 01 * * SUN'
-    
+
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+env:
+  COLUMNS: 120
+
 jobs:
   test_suite:
     name: Pytest on ${{ matrix.python-version }}, ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
Please can you return this to `false` to be consistent with the other holoviz repositories.